thispastwinter/To-Do
# JKA-12: User can create a list and add tasks

## Description

Users can now create a list and add tasks to each respective list.

### 👩‍🔬 Test Instructions

1. NPM 'Start'
2. Default list is "My List"
3. Create a new list via the sidebar. 
4. Add some tasks.
5. Toggle between each list to see the state update!

## 🔎 Reviewer Checklist

> Checked off by the PR **Reviewers**

### Required

> These always need to be checked


>### Roles & Responsibilities
>
>#### 👨‍💻 Justin Klaas
>
>- Initiator of this PR (be sure to set in GitHub UI)
>- Addresses feedback and change requests
>- Merges PR once approved (usually deletes branch unless `develop` or `release`)
>
>#### 👩‍💻 Blake Tarter/Rob LaFeve
>
>- Invited to review PR by **Assignee** (via GitHub UI)
>- Is expected to complete a review and address followup
